engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts.module.js

```diff
@@ -265,7 +265,7 @@ angular.module('Bastion.content-hosts').config(['$stateProvider', function ($sta
         template: '<div ui-view></div>'
     })
     .state('content-hosts.details.errata.index', {
-        url: '/errata?search',
+        url: '/errata?getSearch',
```

Comment:
The reason I changed this to "getSearch" is that is what the URL displayed if I simply typed something into the search and hit enter.

See: https://github.com/Katello/bastion/blob/007321fa37629adf9c462328b5ad5fb5f802c40b/app/assets/javascripts/bastion/components/nutupane.factory.js#L49